### PR TITLE
Allow inabox-rov=1 to exist or not

### PR DESCRIPTION
**Pull request by jonkeller**
Fix red builds
Similar to https://github.com/ampproject/amphtml/pull/14338 but allows the 'inabox-rov=1' string to either exist or not.

### CODE REVIEW

**Comment by jonkeller** [comment]
Feel free to scrap in favor of #14352 if preferred.

**Comment by cvializ** [comment]
If this fixes the flake feel free to rebase and unskip the test. Sorry about the conflict!

File: test/functional/test-error.js [hunk 1]
```diff
@@ -464,7 +464,11 @@ describe('reportErrorToServer', () => {
     const e = user().createError('123');
     const data = getErrorReportData(undefined, undefined, undefined, undefined,
         e, true);
-    expect(data.exps).to.equal('test-exp=1,disabled-exp=0');
+    if (data.exps.indexOf('inabox-rov=1') == -1 {
```
Comment (cvializ) [inline test/functional/test-error.js]: Branching like this feels weird in a test. What causes the test to have `inabox-rov=1` set in one case but unset in another? Is the test non-hermetic?

Comment (jonkeller) [reply test/functional/test-error.js]: Not sure. Was going off of https://github.com/ampproject/amphtml/pull/14338/files which expects it to equal 'inabox-rov=1,test-exp=1,disabled-exp=0' but also that PR's Travis failure in which at least one test has it still equalling 'test-exp=1,disabled-exp=0'
Looks like this originated with https://github.com/ampproject/amphtml/pull/13825/files but I'm puzzled as to why it's only happening now, when 13825 is 4 weeks old.

Comment (cvializ) [reply test/functional/test-error.js]: I would prefer that this test be deterministic and give the same values each time. Barring that, checking explicitly for the individual values would be better than branching.

Comment (jonkeller) [reply test/functional/test-error.js]: Ok. Given that I just created this to get rid of redness, and 14352 achieves that, let's let this one sit until @bradfrizzell returns from vacation.

File: test/functional/test-error.js [hunk 2]
```diff
@@ -455,7 +455,7 @@ describe('reportErrorToServer', () => {
     expect(data.s).to.be.undefined;
   });
 
-  it('should report experiments', () => {
+  it.should('should report experiments', () => {
```
Comment (jridgewell) [inline test/functional/test-error.js]: What's this?

File: test/functional/test-error.js [hunk 3]
```diff
@@ -464,7 +464,11 @@ describe('reportErrorToServer', () => {
     const e = user().createError('123');
     const data = getErrorReportData(undefined, undefined, undefined, undefined,
         e, true);
-    expect(data.exps).to.equal('test-exp=1,disabled-exp=0');
+    if (data.exps.indexOf('inabox-rov=1') == -1) {
+      expect(data.exps).to.equal('test-exp=1,disabled-exp=0');
```
Comment (jridgewell) [inline test/functional/test-error.js]: Why not just do a contains? http://www.chaijs.com/api/bdd/#stringstr-msg

File: test/functional/test-error.js (hunk 2 above)
Comment (jonkeller) [reply test/functional/test-error.js]: That was a typo - didn't address because abandoned the CL and just forgot to close. Will do so now.

**Pull request closed without merging**